templates/waf_web_acl.cfn.yml

```diff
@@ -33,8 +33,30 @@ Resources:
           VisibilityConfig:
             CloudWatchMetricsEnabled: true
             MetricName: "RateLimitRule"
-        - Name: !Sub "${Environment}-${Platform}-XSSRule"
+        - Name: !Sub "${Environment}-${Platform}-APIRateLimitRule"
+          # Throttle API requests more aggressively than general requests
           Priority: 2
+          Action:
+            Block: {}
+          Statement:
+            RateBasedStatement:
+              AggregateKeyType: "IP"
```

Comment:
could this block ourselves?

Reply:
Sure, if we hammer it. But it doesn't blacklist IPs beyond the 60-second window.

The limit is higher now.